pkg/templates/nstemplatetiers/nstemplatetier_generator_test.go

```diff
@@ -140,6 +175,27 @@ func TestCreateOrUpdateResources(t *testing.T) {
 
 			// then
 			require.NoError(t, err)
+			// verify that all TemplateTier CRs were updated
+			tierTmpls = toolchainv1alpha1.TierTemplateList{}
+			err = clt.List(context.TODO(), &tierTmpls, client.InNamespace(namespace))
+			require.NoError(t, err)
+			require.Len(t, tierTmpls.Items, len(testnstemplatetiers.AssetNames())-1) // exclude `metadata.yml` from the AssetNames, it does not result in a TemplateTier resource
+			for _, tierTmpl := range tierTmpls.Items {
+				assert.Equal(t, int64(2), tierTmpl.ObjectMeta.Generation)
+			}
```

Comment:
as I already said above - we should not really update the TierTemplates - they are created once and they should persist as they are. The only resources that are going to be updated ar NSTemplateTier CRs

Reply:
yes, let me refactor that

Reply:
removed in https://github.com/codeready-toolchain/host-operator/commit/c3ce432c266e68faed9d1e1128bb024309c0431b